scripts/globals/status.lua

```diff
@@ -2180,7 +2180,7 @@ dsp.mobMod =
     AUTO_SPIKES         = 46, -- enables additional effect script to process when mob is attacked
     SPAWN_LEASH         = 47, -- forces a mob to not move farther from its spawn than its leash distance
     SHARE_TARGET        = 48, -- mob always targets same target as ID in this var
-    -- 49 Available for use
+    IS_NM               = 49, -- If set, a mob will check as a NM.
```

Comment:
we should make a binding to set MOBTYPE_NOTORIOUS rather than a mobmod.

Reply:
type lives in mobentity.cpp btw

Reply:
Dont think it changes to a true nm stat wise. Its just a check showing as impossible

Reply:
hmm that thing we do about the stats I'm pretty sure is wrong we should stop that and just get their stats right instead of multiplying based on wiki conjecture form over a decade ago.

and these should still inherit other NM like things like not being able to be charmed, rather not add a ton of mobmod checks in the same spots we already have nm checks.

@teschnei https://github.com/DarkstarProject/darkstar/blob/2924080116c08539ab245f2a8947f1d448237bdf/src/map/entities/mobentity.cpp#L529

whadawedo boss?

Reply:
For this PR, I think we should just rename this variable to make it obvious its only for /check, the stats we can deal with later